util/registry/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"gopkg.in/yaml.v1"
+)
+
+func main() {
+	actualRules := getActualRules()
+	statedRules := getStatedRules()
+	missingActualRules := []string{}
+	missingStatedRules := []string{}
+
+	for rule := range actualRules {
+		if _, ok := statedRules[rule]; !ok {
+			missingStatedRules = append(missingStatedRules, rule)
+		}
+	}
+
+	for rule := range statedRules {
+		if _, ok := actualRules[rule]; !ok {
+			missingActualRules = append(missingActualRules, rule)
+		}
+	}
+
+	fmt.Println("Missing Stated Rules: ", missingStatedRules)
+	fmt.Println("Missing Actual Rules: ", missingActualRules)
+}
+
+func getStatedRules() map[string]bool {
+	rules := map[string]bool{}
+	gopath := os.Getenv("GOPATH")
+	registryPath := path.Join(gopath, "src/github.com/codelingo/codelingo/registry/tenets.yaml")
+
+	registryContents, err := ioutil.ReadFile(registryPath)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	r := map[interface{}]map[interface{}]map[interface{}]map[interface{}]interface{}{}
```

Comment:
Fun times :smile:

Reply:
Haha, yup - now that's what I call code quality 😆